### PR TITLE
Adjust commands for creating releases

### PR DESCRIPTION
**Pull request by leo**
Now that we're using a single branch, we need to adjust the release commands accordingly.

### CODE REVIEW

**Review by styfle — APPROVED** [review]
Looks good! We'll also need to update the `diff.js` and `changelog.js` scripts. We probably won't need `diff.js` but we will want a `changelog.js` script for generating release notes.

File: package.json [hunk 1]
```diff
@@ -30,8 +30,8 @@
   "scripts": {
     "lerna": "lerna",
     "bootstrap": "lerna bootstrap",
-    "publish-stable": "git checkout master && git pull && lerna version --exact",
-    "publish-canary": "git checkout canary && git pull && lerna version prerelease --preid canary --exact",
+    "publish-stable": "git pull && lerna version --exact",
```
Comment (styfle) [inline package.json]:
```suggestion
    "publish-stable": "git pull && lerna version --message 'Publish Stable' --exact",
```

File: package.json [hunk 2]
```diff
@@ -30,8 +30,8 @@
...
+    "publish-canary": "git pull && lerna version prerelease --preid canary --exact",
```
Comment (styfle) [inline package.json]:
```suggestion
    "publish-canary": "git pull && lerna version prerelease --preid canary --message 'Publish Canary' --exact",
```

**Comment by leo** [comment]
@styfle Great call! I also adjusted [the docs](https://github.com/zeit/now/wiki/Creating-a-Release).

**Review by styfle — APPROVED** [review]
Perfect 👌

**Comment by styfle** [comment]
@leo You also need to update line 23 of `run.js` to be:

```js
const gitPath = branch === 'master' ? 'HEAD~1' : 'origin/master...HEAD';
```

**Pull request merged**